crates/re_analytics/src/event.rs

```diff
@@ -167,12 +170,40 @@ impl Event for ViewerStarted {
     const NAME: &'static str = "viewer_started";
 }
 
+const RERUN_DOMAINS: [&str; 1] = ["rerun.io"];
+
+/// Given a URL, extract the root domain.
+fn extract_root_domain(url: &str) -> Option<String> {
+    let parsed = Url::parse(url).ok()?;
```

Comment:
Side note: I really dislike how there's no borrowed version of `Url`

Reply:
Ha. Yeah, I was trying for a bit to implement the root-domain extraction as a slice before I realized Url wasn't borrowed.